xml/admin_monitoring_alerting.xml

```diff
@@ -0,0 +1,936 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"  xml:id="monitoring-alerting">
+ <title>Monitoring and Alerting</title>
+ <para>
+  By default, &deepsea; deploys a monitoring and alerting stack on the
+  &smaster;, allowing for multiple instances for a HA setup.
```

Comment:
In SES6 DeepSea no longer deploys monitoring by default, users have to explicitly define a prometheus role for prometheus and alertmanager and the grafana role for grafana.